api/envoy/extensions/filters/http/ext_proc/v3alpha/ext_proc.proto

```diff
@@ -150,9 +150,9 @@ message ExternalProcessor {
   string stat_prefix = 8;
 }
 
-// [#not-implemented-hide:]
 // Extra settings that may be added to per-route configuration for a
-// virtual host or cluster.
+// virtual host or cluster. The name "envoy.filters.http.ext_proc" should
```

Comment:
This comment may be correct now, but it will be wrong once #12274 is fixed.  It might be a good idea to just remove it, to avoid having to remember to update it later.

Reply:
Sure -- I took that sentence out. 

FWIW, every time I change a .proto and have to run "proto_format fix" it ends up wanting to change a number of other files and running it has become quite painful. Perhaps we can figure out why it's less consistent than it used to be and fix that.